prelude/__internal__/Either.mad

```diff
@@ -66,8 +69,7 @@ instance Monad (Either e) {
 
 
 /**
- * The Show instance of Maybe returns Right or Left, following by the showed
- * value it contains.
+ * Use `show` to transform an Either to a string.
```

Comment:
Not sure where to comment on that, but I like the idea of having markdown in there.

Reply:
Documented as an issue here: https://github.com/madlib-lang/madlib/issues/15